src/layers/legacy/medCoreLegacy/gui/medAbstractWorkspaceLegacy.cpp

```diff
@@ -697,16 +697,18 @@ QWidget* medAbstractWorkspaceLegacy::buildViewLinkMenu()
 
     // build a stringList witth all the param names
     foreach(medAbstractParameterL* viewParam, viewsParams)
```

Comment:
for :
not needed but... ;)

Reply:
that's not in this PR on 3.1.1 but it's already done on the master

Reply:
;)